Tasks/task3-4.py

```diff
@@ -0,0 +1,15 @@
+# Напишите программу, которая будет преобразовывать десятичное число в двоичное.
+
+# 45 -> 101101
+# 3 -> 11
+# 2 -> 10
+
+dex = input('Введите десятичное число: ')
+temp = int(dex)
+bin = ''
+while temp > 0:
+    x = int(temp % 2)
```

Comment:
```suggestion
    x = temp % 2
```
Результат такой операции всегда будет целым. Если, конечно, вы не делите с остатком на единицу